Find a functionally equivalent code:com.android.internal.view.menu.MenuBuilder.addMenuPresenter:COMMENT
Method Modifier: public      internal    
Comment:/**
 * Add a presenter to this menu. This will only hold a WeakReference;
 * you do not need to explicitly remove a presenter, but you can using
 * {@link #removeMenuPresenter(MenuPresenter)}.
 *
 * @param presenter The presenter to add
 */

@@ -1,5 +1,3 @@
 {
-    mPresenters.add(new WeakReference<MenuPresenter>(presenter));
-    presenter.initForMenu(mContext, this);
-    mIsActionItemsStale = true;
+    addMenuPresenter(presenter, mContext);
 }

